section4/reflection.md

```diff
@@ -2,21 +2,55 @@
+    * I felt bogged down right around Session 5 on despite having had some momentum, though life intervened a few times and got me to the left of how far ahead I'd envisioned staying throughout mod 0 from when I started
+    * This malaise made it hard to do timers or breaks or anything, I just needed to power through and so that is what I did / am doing
 
-1. In your own words, what is a Class?
+2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
 
-1. What is an attribute of a Class?
+    * Yes, not having some critcal systems set up in your life before you start having all this to keep up with can be a real drag on progress, that seemed likely true to me before I started mod 0 of course but it especially really does now
 
-1. What is behavior of a Class?
+3. In your own words, what is a Class?
 
-1. In the space below, create a Dog class with at least 2 attributes and 2 behaviors:
+    * a Class is a pre-defined template represented set from which to spawn an object belonging to the set keeping that object Class-consistent
+
+4. What is an attribute of a Class?
+
+    * attributes as part of a Class are specified by instance variables denoted by "@-" prefix to the name 
+
+5. What is behavior of a Class?
+
+    * a Class may be granted behaviors by specifying a Class or instance method nested in the Class definition 
+
+6. In the space below, create a Dog class with at least 2 attributes and 2 behaviors:
 
 ```rb
 
+class Dog
+  
+  def initialize(name, breed)
+    @name = name
+    @breed = breed
+  end
+  
+  def bark
+  end
+
+  def poo
+  end
+
+end
 
 ```
 
-1. How do you create an instance of a class?
+7. How do you create an instance of a class?
+
+    * Select a variable name to store the instance in, then set it equal to YourClass.new
+    * This creates an instance of the class with the selected name
+
+8. What questions do you still have about classes in Ruby?
 
-1. What questions do you still have about classes in Ruby?
\ No newline at end of file
+    * As in my burrito_broaden.rb file I'm not sure about results I was getting attempting to do things inside class (instance) methods
+    * I'm still not clear at all at how the syntax in particular conveys the ideas expressed in the technical language about what the code is doing
+    * If someone were to walk me thorough how all of this particular syntax expresses the underlying structure of class and variables and methods accessible at various levels I imagine I'd understand more clearly
```

Comment:
Hopefully my explanations help out a bit with this. You're doing great, and you're right on track with figuring this stuff out, and I appreciate the amount of effort you're putting in. Keep up the great work in Mod 1 🙂